pkgs/development/python-modules/imapclient/default.nix

```diff
@@ -3,23 +3,24 @@
 , fetchFromGitHub
 , six
 , pytestCheckHook
-, mock
 }:
 
 buildPythonPackage rec {
   pname = "imapclient";
-  version = "2.2.0";
+  version = "2.3.1";
+
+  format = "setuptools";
```

Comment:
Ist `setuptools` the default format? Is this required?

Reply:
We might change/remove the default in the future, that's why it makes sense to add it.